app/serializers/claim_base_serializer.rb

```diff
@@ -1,5 +1,33 @@
 # frozen_string_literal: true
 class ClaimBaseSerializer < ActiveModel::Serializer
-  attributes :id, :date_filed, :min_est_date, :max_est_date,
+  attributes :id, :evss_id, :date_filed, :min_est_date, :max_est_date,
              :tracked_items, :phase_dates, :open, :waiver_submitted
+
+  def date_filed
+    object.data['date']
```

Comment:
I preferred how in the old code, there was a central function that converted things from a JSON object to Ruby native. Is there a way to do something like that here so that the simple string to string attributes all get mapped in one place and the more complicated attributes go through individual methods? Or does everything need to have an individual method like this?